submissions/Roophee/JS - DOM/index.js

```diff
@@ -0,0 +1,111 @@
+"use strict";
+
+const clickBurger = document.querySelector(".burger");
+const clickContent = document.querySelector(".content");
+const clickHeader = document.querySelector(".header");
+const clickfooter = document.querySelector(".footer");
+
+const deBar = document.querySelector(".aside");
+
+
+// MOVE ASIDE & CONTENT
+
+function hideSideBar(e) { 
+    e.preventDefault();
+    deBar.classList.toggle('hide__sidebar');
+    clickContent.classList.toggle('move_content');
+    clickBurger.classList.toggle('burger__stop');
+    console.log(e.type)
+
+};
+
+clickBurger.addEventListener('click', hideSideBar);
+
+// Enter-key event show-hide aside
+
+clickBurger.addEventListener('keydown', function (e) {
+    e.preventDefault();
+    if (e.keyCode == "13") {
+        deBar.classList.toggle('hide__sidebar');
+        clickContent.classList.toggle('move_content');
+        clickBurger.classList.toggle('burger__stop');
+    }
+}
+);
+
+// CLOSE ASIDE WHEN MOUSE CLICK OUT
+
+function hideSideBarClickAway(e) {
+    e.preventDefault();
+    for (let i = 0; i < deBar.classList.length; i++) {
+        if (deBar.classList[i] === 'hide__sidebar') {
+            deBar.classList.toggle('hide__sidebar');
+            clickBurger.classList.toggle('burger__stop');
+            clickContent.classList.toggle('move_content');
+        };
+    };
+};
+
+clickContent.addEventListener('click', hideSideBarClickAway);
+clickHeader.addEventListener('click', hideSideBarClickAway);
+clickfooter.addEventListener('click', hideSideBarClickAway);
+
+// ADD CONTENT IN .content
+
+function addContentOnClick() {
+    clickContent.innerHTML = "";
+
+    let contentRaw = contentStorage[this.innerText.toLowerCase()];
+    let makeContent = `<div class="content__header">
+    <h1 name="top">
+    ${contentRaw.title.toUpperCase()}
```

Comment:
visual representing must be implemented in the CSS file.

Reply:
What it mean? In CSS file I describe how to display elements with classes assigned to them.  Please, explain in more detail.

Reply:
Why you use this method(.toLowerCase() line 58 and .toUpperCase() line 61)?

Reply:
Ok. I understend, but say to me how it will be corretc to set. 
line58)  my .nav__link contains <span> whith text? that shoud be a lebel for buttins in <aside>. Text in span starts with capital leter. Thets why I use .toLowerCase() . Then I text this text and use them to apply like property of DataStorage.

line61) Understend, I can avoid  call this method.

Reply:
There a couple of ways to fix it:
1) Fix/transform content in your data object.
2) If you can't control incoming data for manipulating by the first letter of words use CSS pseudoelement `::first-letter`. Read more(https://developer.mozilla.org/en-US/docs/Web/CSS/::first-letter)

Reply:
You mean that all text with the exception of content should be lowerCase and  decoration and transforming I should make only with help of CSS.
Did I understand you correctly?

Reply:
If you have access to the incoming data(yes you have) you transform it into what you need: content in corresponding fields into content with corresponding letter case.
Avoid unnecessary manipulation with data.
Use CSS as insurance for an unexpected situation.
If you want to discuss it in Russian/Ukrainian ping me in telegram @A_Ostrovnyy.)